src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/catchparametername/Example2.java

```diff
@@ -8,19 +8,25 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.naming.catchparametername;
+
+
+import java.io.IOException;
+
 // xdoc section -- start
-public class MyTest {
+public class Example2 {
```

Comment:
Please make both code examples exactly the same except for class names and comments so that it is easy for users to spot the difference in check behavior.

Reply:
Used the same code in the both example file.